doc/schemas/listpeers.schema.json

```diff
@@ -235,6 +235,10 @@
                   "type": "txid",
                   "description": "ID of the funding transaction"
                 },
+	        "funding_outnum": {
+		  "type": "u32",
+                  "description": "The 0-based output number of the funding transaction which opens the channel"
+		},
```

Comment:
Mh from CI there is a formatting problem here, maybe a `make check-doc` will call `jq` to auto formatting the json file

Reply:
Yep that was the issue for this test. Nice catch!